code/__HELPERS/roundend.dm

```diff
@@ -238,6 +235,8 @@
 	//Set news report and mode result
 	mode.set_round_result()
 
+	to_chat(world, "<span class='infoplain'><BR><BR><BR><span class='big bold'>The round has ended.</span></span>")
```

Comment:
```suggestion
	to_chat(world, span_infoplain(span_big(span_bold("<BR><BR><BR>The round has ended."))))
```

yes i know the line breaks weren't in the `big bold` previously but they're line breaks it doesn't matter. much cleaner to my eyes

Reply:
I cannot believe, cannot BELIEVE you are making me do CODE changes. in MY lazily made PR? How COULD you.

Reply:
bro i already did all of the hard work just press the commit suggestion button